apps/team-awesome-web/src/components/forms/filter/Filter.styled.ts

```diff
@@ -0,0 +1,40 @@
+import styled from "styled-components";
+
+export const Select = styled.select`
+  background-color: ${({ theme }) => theme.grey1};
+  border-radius: 4px;
+  border: solid 1px ${({ theme }) => theme.grey3};
+  color: ${({ theme }) => theme.grey6};
+  display: grid;
+  grid-auto-flow: row;
+  justify-self: end;
+  min-width: 120px;
+  padding: 16px 14px;
+
+  &::placeholder {
+    opacity: 0.5;
+  }
+
+  &:hover {
+    border: solid 1px ${({ theme }) => theme.grey5};
+    cursor: pointer;
+  }
+
+  &:active,
+  &:focus {
+    border: solid 1px ${({ theme }) => theme.grey6};
+    cursor: initial;
+  }
+`;
+
+export const Option = styled.option``;
```

Comment:
Since there's no additional styling, I would delete this styled component and simply use `<option>`.